sdk/src/gateway/tokens.ts

```diff
@@ -58,6 +58,18 @@ const bobTokens = [
         },
         logoURI: 'https://raw.githubusercontent.com/bob-collective/bob/master/assets/uniBTC.svg',
     },
+    // FIXME update with mainnet deployment results
+    {
+        name: 'Bob BTC',
+        symbol: 'bobBTC',
+        decimals: 8,
+        tokens: {
+            bob: {
+                address: '0x4496ebE7C8666a8103713EE6e0c08cA0cD25b888',
+            },
+        },
+        logoURI: 'https://raw.githubusercontent.com/bob-collective/bob/master/assets/bobBTC.svg',
+    },
```

Comment:
I’d suggest removing this for now or setting it to `NULL`, since the offramp will initially support only WBTC and TBTC, and there’s no `bobBTC` available in near future yet.